api/GPUSupportedFeatures.json

```diff
@@ -113,6 +113,715 @@
           }
         }
       },
+      "feature_bgra8unorm-storage": {
+        "__compat": {
+          "description": "<code>bgra8unorm-storage</code> feature",
+          "spec_url": "https://gpuweb.github.io/gpuweb/#bgra8unorm-storage",
+          "tags": [
+            "web-features:webgpu"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "113"
+            },
+            "chrome_android": {
+              "version_added": "121"
+            },
+            "deno": [
+              {
+                "version_added": "1.39",
+                "flags": [
+                  {
+                    "type": "runtime_flag",
+                    "name": "--unstable-webgpu"
+                  }
+                ]
+              },
+              {
+                "version_added": "1.8",
+                "version_removed": "1.32"
+              }
+            ],
```

Comment:
How did you determine the Deno support status for the new `GPUSupportedFeatures` subfeatures?